uavcan/file/Path.1.0.uavcan

```diff
@@ -0,0 +1,10 @@
+#
+# Nested type.
+# A file system path encoded in UTF8.
+# The only valid separator is the forward slash "/".
+# A single slash ("/") refers to the root directory.
+#
+
+uint8 SEPARATOR = '/'
+
+uint8[<=200] path
```

Comment:
I was seriously considering to reduce it because 200 bytes is a lot of space and we're taking away from the payload in the `file.Write` service. With this change, `file.Write` would be almost 400 bytes large, quite taxing for small nodes. Do we really want that? Especially considering that:

* The length field will be 9 bit wide
* Who cares about Windows